evap/static/ts/src/infobox.ts

```diff
@@ -17,6 +18,8 @@ export class InfoboxLogic {
         // close the infobox and save state
         this.closeButton.addEventListener("click", event => {
             this.infobox.classList.add("closing");
+            this.infobox.classList.remove("opening");
+            clearTimeout(this.timeout);
```

Comment:
Should we reset `this.timeout` afterwards? Also in the case the timeout callback is triggered normally? Or do we not care?

Reply:
clearTimeout ignores invalid timeout parameters, so  I think it should be fine